src/data/sample-new-sections-sequence-te.json

```diff
@@ -0,0 +1,273 @@
+{
+  "abstract": "",
+  "description": "",
+  "display_title": "",
+  "logo": "",
+  "runtime": "Activity Player",
+  "theme_id": null,
+  "thumbnail_url": "",
+  "title": "TE Test Sequence",
+  "project": null,
+  "activities": [
+    {
+      "description": "<p>This is a test activity which is contained within a sequence and uses the teacher edition plugin.</p>",
+      "editor_mode": 0,
+      "id": 21123,
+      "layout": 0,
+      "name": "TE Test 1",
+      "notes": "",
+      "related": "",
+      "runtime": "Activity Player",
+      "show_submit_button": true,
+      "student_report_enabled": true,
+      "thumbnail_url": "",
+      "time_to_complete": null,
+      "version": 2,
+      "theme_name": null,
+      "project": null,
+      "pages": [
+        {
+          "id": 309275,
+          "is_completion": false,
+          "is_hidden": false,
+          "name": null,
+          "position": 1,
+          "show_sidebar": false,
+          "sidebar": null,
+          "sidebar_title": "Did you know?",
+          "sections": [
+            {
+              "layout": "l-6040",
+              "is_hidden": false,
+              "secondary_display_mode": "stacked",
+              "secondary_column_collapsible": false,
```

Comment:
This is a note to @emcelroy and I to verify that when we send this property out from LARA, it has the same name.